interface/package.json

```diff
@@ -24,18 +24,18 @@
     "@alova/adapter-xhr": "2.0.11",
     "@emotion/react": "^11.13.5",
     "@emotion/styled": "^11.13.5",
-    "@mui/icons-material": "^6.1.9",
-    "@mui/material": "^6.1.9",
+    "@mui/icons-material": "^6.1.10",
+    "@mui/material": "^6.1.10",
     "@table-library/react-table-library": "4.1.7",
     "alova": "3.2.6",
     "async-validator": "^4.2.5",
     "jwt-decode": "^4.0.0",
     "mime-types": "^2.1.35",
     "preact": "^10.25.1",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
-    "react-icons": "^5.3.0",
-    "react-router": "^7.0.1",
+    "react": "^19.0.0",
```

Comment:
react 19 is a big change, we'll need to keep an eye out that nothing is broken downstream

Reply:
Yes, i quickly navigated all pages and it looks ok. 
Only thing is recognized, but this was also before, the router tabs in security and network are not highlighted/underlined as in older versions.
Acccording to upgrade guide https://react.dev/blog/2024/04/25/react-19-upgrade-guide most breaking changes should also give a warning.
If i see something strange,i will also test with older react versions.